[CI] HistoBackedRangeAggregatorTests testLargerRangesAccuracy failing

**Build scan:**
https://gradle-enterprise.elastic.co/s/cqimgh6lqkfri/tests/:x-pack:plugin:analytics:test/org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests/testLargerRangesAccuracy

**Reproduction line:**
```
./gradlew ':x-pack:plugin:analytics:test' --tests "org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testLargerRangesAccuracy" -Dtests.seed=D50926F21C429E04 -Dtests.locale=es-CR -Dtests.timezone=NST -Druntime.java=20
```

**Applicable branches:**
main

**Reproduces locally?:**
Yes

**Failure history:**
https://gradle-enterprise.elastic.co/scans/tests?tests.container=org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests&tests.test=testLargerRangesAccuracy

**Failure excerpt:**
```
java.lang.RuntimeException: java.util.concurrent.ExecutionException: org.elasticsearch.common.breaker.CircuitBreakingException: cranky breaker

  at __randomizedtesting.SeedInfo.seed([D50926F21C429E04:8BA02CF824830F7D]:0)
  at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:730)
  at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:688)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:588)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.runWithCrankyCircuitBreaker(AggregatorTestCase.java:489)
  at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:472)
  at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testRanges(HistoBackedRangeAggregatorTests.java:178)
  at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testLargerRangesAccuracy(HistoBackedRangeAggregatorTests.java:128)
  at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
  at java.lang.reflect.Method.invoke(Method.java:578)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
  at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
  at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
  at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
  at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
  at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
  at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
  at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
  at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
  at java.lang.Thread.run(Thread.java:1623)

  Caused by: java.util.concurrent.ExecutionException: org.elasticsearch.common.breaker.CircuitBreakingException: cranky breaker

    at java.util.concurrent.FutureTask.report(FutureTask.java:122)
    at java.util.concurrent.FutureTask.get(FutureTask.java:191)
    at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:726)
    at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:688)
    at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:588)
    at org.elasticsearch.search.aggregations.AggregatorTestCase.runWithCrankyCircuitBreaker(AggregatorTestCase.java:489)
    at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:472)
    at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testRanges(HistoBackedRangeAggregatorTests.java:178)
    at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testLargerRangesAccuracy(HistoBackedRangeAggregatorTests.java:128)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
    at java.lang.reflect.Method.invoke(Method.java:578)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
    at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
    at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
    at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
    at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
    at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
    at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
    at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
    at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
    at java.lang.Thread.run(Thread.java:1623)

    Caused by: org.elasticsearch.common.breaker.CircuitBreakingException: cranky breaker

      at org.elasticsearch.indices.CrankyCircuitBreakerService$1.addEstimateBytesAndMaybeBreak(CrankyCircuitBreakerService.java:38)
      at org.elasticsearch.common.util.BigArrays.adjustBreaker(BigArrays.java:466)
      at org.elasticsearch.common.util.BigArrays.validate(BigArrays.java:524)
      at org.elasticsearch.common.util.BigArrays.newLongArray(BigArrays.java:677)
      at org.elasticsearch.common.util.MockBigArrays.newLongArray(MockBigArrays.java:226)
      at org.elasticsearch.common.util.BigArrays.resize(BigArrays.java:695)
      at org.elasticsearch.common.util.MockBigArrays.resize(MockBigArrays.java:237)
      at org.elasticsearch.common.util.BigArrays.grow(BigArrays.java:709)
      at org.elasticsearch.search.aggregations.bucket.BucketsAggregator.grow(BucketsAggregator.java:72)
      at org.elasticsearch.search.aggregations.bucket.BucketsAggregator.collectBucket(BucketsAggregator.java:79)
      at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregator$NoOverlap.collect(HistoBackedRangeAggregator.java:191)
      at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregator$1.collect(HistoBackedRangeAggregator.java:137)
      at org.elasticsearch.search.aggregations.LeafBucketCollector.collect(LeafBucketCollector.java:86)
      at org.apache.lucene.tests.search.AssertingLeafCollector.collect(AssertingLeafCollector.java:52)
      at org.apache.lucene.tests.search.AssertingCollector$1.collect(AssertingCollector.java:66)
      at org.apache.lucene.tests.search.AssertingLeafCollector.collect(AssertingLeafCollector.java:52)
      at org.apache.lucene.tests.search.AssertingLeafCollector.collect(AssertingLeafCollector.java:52)
      at org.apache.lucene.search.Weight$DefaultBulkScorer.scoreRange(Weight.java:274)
      at org.apache.lucene.search.Weight$DefaultBulkScorer.score(Weight.java:254)
      at org.apache.lucene.tests.search.AssertingBulkScorer.score(AssertingBulkScorer.java:101)
      at org.apache.lucene.search.TimeLimitingBulkScorer.score(TimeLimitingBulkScorer.java:82)
      at org.apache.lucene.search.BulkScorer.score(BulkScorer.java:38)
      at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:776)
      at org.apache.lucene.tests.search.AssertingIndexSearcher.search(AssertingIndexSearcher.java:78)
      at org.apache.lucene.search.IndexSearcher.lambda$search$1(IndexSearcher.java:715)
      at java.util.concurrent.FutureTask.run(FutureTask.java:317)
      at org.apache.lucene.search.SliceExecutor.invokeAll(SliceExecutor.java:40)
      at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:722)
      at org.apache.lucene.search.IndexSearcher.search(IndexSearcher.java:688)
      at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:588)
      at org.elasticsearch.search.aggregations.AggregatorTestCase.runWithCrankyCircuitBreaker(AggregatorTestCase.java:489)
      at org.elasticsearch.search.aggregations.AggregatorTestCase.searchAndReduce(AggregatorTestCase.java:472)
      at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testRanges(HistoBackedRangeAggregatorTests.java:178)
      at org.elasticsearch.xpack.analytics.aggregations.bucket.range.HistoBackedRangeAggregatorTests.testLargerRangesAccuracy(HistoBackedRangeAggregatorTests.java:128)
      at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
      at java.lang.reflect.Method.invoke(Method.java:578)
      at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at org.apache.lucene.tests.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:48)
      at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
      at org.apache.lucene.tests.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:45)
      at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
      at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
      at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:843)
      at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:490)
      at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
      at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
      at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at org.apache.lucene.tests.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:38)
      at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
      at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at org.apache.lucene.tests.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
      at org.apache.lucene.tests.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:43)
      at org.apache.lucene.tests.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:44)
      at org.apache.lucene.tests.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:60)
      at org.apache.lucene.tests.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:47)
      at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
      at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:390)
      at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:850)
      at java.lang.Thread.run(Thread.java:1623)

```

--- a/x-pack/plugin/analytics/src/test/java/org/elasticsearch/xpack/analytics/aggregations/bucket/range/HistoBackedRangeAggregatorTests.java
+++ b/x-pack/plugin/analytics/src/test/java/org/elasticsearch/xpack/analytics/aggregations/bucket/range/HistoBackedRangeAggregatorTests.java
@@ -8,6 +8,7 @@
 package org.elasticsearch.xpack.analytics.aggregations.bucket.range;
 
 import org.apache.lucene.document.DoubleDocValuesField;
+import org.apache.lucene.index.DirectoryReader;
 import org.apache.lucene.index.IndexReader;
 import org.apache.lucene.search.IndexSearcher;
 import org.apache.lucene.store.Directory;
@@ -49,8 +50,10 @@ public class HistoBackedRangeAggregatorTests extends AggregatorTestCase {
         long absError = 0L;
         long docCount = 0L;
         for (int k = 0; k < 10; k++) {
-            try (Directory dir = newDirectory(); RandomIndexWriter w = new RandomIndexWriter(random(), dir)) {
-                docCount += generateDocs(w);
+            try (Directory dir = newDirectory()) {
+                try (RandomIndexWriter w = new RandomIndexWriter(random(), dir)) {
+                    docCount += generateDocs(w);
+                }
                 double[] steps = IntStream.range(2, 99).filter(i -> i % 2 == 0).mapToDouble(Double::valueOf).toArray();
 
                 PercentilesAggregationBuilder rawPercentiles = new PercentilesAggregationBuilder("my_agg").field(RAW_FIELD_NAME)
@@ -61,8 +64,8 @@ public class HistoBackedRangeAggregatorTests extends AggregatorTestCase {
                     .percentilesConfig(new PercentilesConfig.Hdr())
                     .percentiles(steps);
 
-                try (IndexReader reader = w.getReader()) {
-                    IndexSearcher searcher = newSearcher(reader);
+                try (DirectoryReader reader = DirectoryReader.open(dir)) {
+                    IndexSearcher searcher = newIndexSearcher(reader);
                     RangeAggregationBuilder aggBuilder = new RangeAggregationBuilder("my_agg").field(HISTO_FIELD_NAME);
 
                     RangeAggregationBuilder rawFieldAgg = new RangeAggregationBuilder("my_agg").field(RAW_FIELD_NAME);
@@ -119,7 +122,6 @@ public class HistoBackedRangeAggregatorTests extends AggregatorTestCase {
         testRanges(ranges, "manual_medium_ranges");
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/98713")
     public void testLargerRangesAccuracy() throws Exception {
         List<RangeAggregator.Range> ranges = Arrays.asList(
             new RangeAggregator.Range(null, null, 8.0),
@@ -164,11 +166,12 @@ public class HistoBackedRangeAggregatorTests extends AggregatorTestCase {
         long absError = 0L;
         long docCount = 0L;
         for (int k = 0; k < 10; k++) {
-            try (Directory dir = newDirectory(); RandomIndexWriter w = new RandomIndexWriter(random(), dir)) {
-                docCount += generateDocs(w);
-
-                try (IndexReader reader = w.getReader()) {
-                    IndexSearcher searcher = newSearcher(reader);
+            try (Directory dir = newDirectory()) {
+                try (RandomIndexWriter w = new RandomIndexWriter(random(), dir)) {
+                    docCount += generateDocs(w);
+                }
+                try (DirectoryReader reader = DirectoryReader.open(dir)) {
+                    IndexSearcher searcher = newIndexSearcher(reader);
                     RangeAggregationBuilder aggBuilder = new RangeAggregationBuilder("my_agg").field(HISTO_FIELD_NAME);
                     RangeAggregationBuilder rawFieldAgg = new RangeAggregationBuilder("my_agg").field(RAW_FIELD_NAME);
                     ranges.forEach(r -> {

